lib/core/src/server/utils/load-custom-babel-config.js

```diff
@@ -46,15 +46,21 @@ function loadFromPath(babelConfigPath) {
   return config;
 }
 
-function isBabelLoader8() {
+function isBabelLoader8(options) {
```

Comment:
that's very hacky hack =( core should be agnostic to whatever app is using it... I think we need to override this in the sb/react itself

Reply:
I agree that this would be better handled in `@storybook/react`, but I didn't see a way to do that. I don't agree this is a hack though, as an `is` function can certainly take arguments.

I'm still new to the project, and would be very open to suggestions on how to improve this if you can point me to the right place.